packages/core/src/presentation/forms/validation/validators.ts

```diff
@@ -9,13 +9,31 @@ const VALID_EMAIL_REGEX = new RegExp(
   '^(([^<>()\\[\\]\\\\.,;:\\s@"]+(\\.[^<>()\\[\\]\\\\.,;:\\s@"]+)*)|(".+"))@((\\[[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}])|(([a-zA-Z\\-0-9]+\\.)+[a-zA-Z]{2,}))$',
 );
 
+const VALID_URL = new RegExp('^https?://.+$');
+
+const VALID_CDEVENT_REGEX = new RegExp('^dev\\.cdevents\\.[^.]+\\.[^.]+$');
```

Comment:
It would also be good to make this configurable from `settings.js` - some enterprises might choose a different type schema.